UnityProject/Assets/Scripts/Core/Editor/MiscFunctions_RRT.cs

```diff
@@ -38,361 +39,28 @@ public class MiscFunctions_RRT : EditorWindow
-
-		[MenuItem("Tools/StartAssetEditing")]
-		public static void StartAssetEditing()
-		{
-			AssetDatabase.StartAssetEditing();
-		}
-
-		[MenuItem("Tools/Sprites/SetUpClothingSprites")]
-		public static void SetSpriteDataForClothing()
-		{
-			AssetDatabase.StartAssetEditing();
-			var ClothingPefabs = LoadAllPrefabsOfType<ClothingV2>(Application.dataPath);
-			foreach (var ClothingPefab in ClothingPefabs)
-			{
-				var DataToUse = ClothingPefab.CurrentClothData;
-				if (DataToUse == null) continue;
-				var Data = ClothingPefab.GenItemsSprites(DataToUse);
-				if (Data == null) continue;
-				var att = ClothingPefab.GetComponent<ItemAttributesV2>();
-				att.SetSprites(Data);
-
-				EditorUtility.SetDirty(ClothingPefab.gameObject);
-				Undo.RecordObject(ClothingPefab.gameObject, "set SetSprites");
-			}
-			AssetDatabase.StopAssetEditing();
-			AssetDatabase.SaveAssets();
-		}
-
-
-		[MenuItem("Tools/Sprites/Convert Json Sprites")]
-		public static void ConvertJsonSprites()
-		{
-			spriteCatalogue =
-				AssetDatabase.LoadAssetAtPath<SpriteCatalogue>(
-					"Assets/Resources/ScriptableObjectsSingletons/SpriteCatalogueSingleton.asset");
-			ToSeve.Clear();
-			ToDel.Clear();
-			DirSearch_ex3(Application.dataPath + "/SpriteJsonToSO");
-
-			foreach (var oDe in ToDel)
-			{
-				AssetDatabase.DeleteAsset(oDe);
-			}
-
-			var AAA = FindAssetsByType<SpriteCatalogue>();
-			foreach (var Seve in ToSeve)
-			{
-				AssetDatabase.CreateAsset(Seve.Value, Seve.Key);
-				AAA[0].AddToCatalogue(Seve.Value);
-			}
-
-			ToSeve.Clear();
-			ToDel.Clear();
-			EditorUtility.SetDirty(AAA[0]);
-			AssetDatabase.SaveAssets();
-		}
-
-		[MenuItem("Tools/Sprites/Reset SO index")]
-		public static void Reset()
-		{
-			AssetDatabase.StartAssetEditing();
-			var AAA = FindAssetsByType<SpriteCatalogue>();
-			foreach (var AA in AAA)
-			{
-				AA.Catalogue = new List<SpriteDataSO>();
-				EditorUtility.SetDirty(AA);
-			}
-
-			var SOs = FindAssetsByType<SpriteDataSO>();
-			foreach (var SO in SOs)
-			{
-				AAA[0].AddToCatalogue(SO);
-			}
-
-			EditorUtility.SetDirty(AAA[0]);
-			AssetDatabase.StopAssetEditing();
-			AssetDatabase.SaveAssets();
-		}
-
-		[MenuItem("Tools/Debug/CheckURLHubValidation")]
-		public static async Task Dothing()
-		{
-			var data = await HubValidation.RequestOpenURL(new Uri("https://old.reddit.com"), " because lol ", false);
-			data = await HubValidation.RequestAPIURL(new Uri("https://old.reddit.com"), " Because I needed ", false);
-			data = await HubValidation.RequestTrustedMode("AAAAAAAAAAAAAAAA");
-			Loggy.LogError(data.ToString());
-		}
-
-		public class coolClass
-		{
-			public int val = 0;
-		}
-
-		class LoginBody
-		{
-			public string email { get; set; }
-			public string password { get; set; }
-		}
-
-
 
 		[MenuItem("Tools/Debug/------------ Debug function -----------")]
 		public static void Generate()
 		{
-
-			var coolsdsa = AllowedReflection.GetTypeByName("LightSource");
+			// AssetDatabase.StartAssetEditing();
```

Comment:
remove code that isn't used

Reply:
well has useful code snippets

Reply:
move it to another function then

Reply:
... is a debug thing

Reply:
still terrible code quality

Reply:
It's literally buried in a debug function xD

Reply:
then remove it?

Reply:
this is just a wall of useless text when I look at the whole file

Reply:
it's useful for debugging stuff, this second copy and paste existing code

Reply:
it's full of debug functions xD

Reply:
functions that aren't being used because they're commented out

Reply:
yeah, uncommon them and copy the code you need

Reply:
at least put it at the very bottom of the script if you really want to keep it in

Reply:
well I'm putting it a top so I can find the function quicker

Reply:
![head-rub-rub](https://github.com/unitystation/unitystation/assets/34368774/98f997b8-8b88-4735-9520-4caf986262fd)

we're going in circles
I'm just going to approve this and move on